UnityProject/Assets/Scripts/Health/Living/LivingHealthBehaviour.cs

```diff
@@ -694,6 +694,87 @@ public void UpdateClientBodyPartStats(BodyPartType bodyPartType, float bruteDama
 		}
 	}
 
+	/// ---------------------------
+	/// Electrocution Methods
+	/// ---------------------------
+	/// Note: Electrocution for players is extended in PlayerHealth deriviative.
+	/// This is a generic electrocution implementation that just deals damage.
+
+	/// <summary>
+	/// Electrocutes a mob, applying damage to the victim depending on the electrocution power.
+	/// </summary>
+	/// <param name="electrocution">The object containing all information for this electrocution</param>
+	/// <returns>Returns an ElectrocutionSeverity for when the following logic depends on the elctrocution severity.</returns>
+	public virtual ElectrocutionSeverity Electrocute(Electrocution electrocution)
+	{
+		float resistance = ApproximateElectricalResistance(electrocution.Voltage);
+		electrocution.shockPower = Electrocution.CalculateShockPower(electrocution.Voltage, resistance);
+		SetElectrocutionSeverity(electrocution);
+
+		switch (electrocution.severity)
+		{
+			case ElectrocutionSeverity.None:
+				break;
+			case ElectrocutionSeverity.Mild:
+				MildElectrocution(electrocution);
+				break;
+			case ElectrocutionSeverity.Painful:
+				PainfulElectrocution(electrocution);
+				break;
+			case ElectrocutionSeverity.Lethal:
+				LethalElectrocution(electrocution);
+				break;
+		}
+
+		return electrocution.severity;
+	}
+
+	/// <summary>
+	/// Finds the severity of the electrocution.
+	/// In the future, this would depend on the victim's size. For now, assume humanoid size.
+	/// </summary>
+	/// <param name="electrocution">The object containing all information for this electrocution</param>
+	/// <returns>Severity enumerable - None, Mild, Painful, Lethal</returns>
+	protected virtual ElectrocutionSeverity SetElectrocutionSeverity(Electrocution electrocution)
```

Comment:
If anything this should just be a "GetElectrocutionSeverity", it should not modify Electrocution as a side effect. Avoiding side effects and state mutation where possible helps keep complexity down.